cmd/credential.go

```diff
@@ -30,12 +30,13 @@ const (
 	accessKeyMinLen = 5
 	accessKeyMaxLen = 20
 	secretKeyMinLen = 8
-	secretKeyMaxLen = 100
 
 	alphaNumericTable    = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
 	alphaNumericTableLen = byte(len(alphaNumericTable))
 )
 
+var secretKeyMaxLen = 40
```

Comment:
- we should use different constants for Azure and AWS S3 maximum secret key/access key length instead of literals.
- as per our convention, we should call the above variable globalSecretKeyMaxLen.